roles/hosts/tasks/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: create host entries
+  lineinfile:
+    destination: /etc/hosts
+    line: "{{ item.ip }} {{ item.name }}"
+    when: hosts is define
+    with_items:
+      - host_list
```

Comment:
This is a generic role, but I think this is intended for at least the hypervisor node. Correct?

Reply:
It's incredibly generic and just meant to be boilerplate so when we need to implement it, we're not writing it all from scratch.